packages/real/crates/real-base/Cargo.toml

```diff
@@ -18,6 +18,7 @@ wasm = [ "dep:wasm-bindgen" ]
 
 [dependencies]
 thiserror = "1.0"
+bitflags = "2.4"
```

Comment:
Should this be rather optional and only enabled as a feature, when `wasm` feature is on?

Reply:
ok, done